camera/ios/Plugin/CameraPlugin.swift

```diff
@@ -138,6 +140,18 @@ public class CameraPlugin: CAPPlugin {
             settings.presentationStyle = .fullScreen
         }
 
+        // FarmQA
+        settings.createThumbnail = call.getBool("createThumbnail") ?? false
+        settings.thumbnailHeight = CGFloat(call.getInt("thumbnailHeight") ?? 0)
```

Comment:
The Android version initializes these values differently. The height & width defaults to 70 if they are not set.
The createThumbnail is not set to true based on the height and width.

Reply:
I'm fine with changing the Android version. It  just needs to be consistent.